dashboard/src/features/accounts/components/AccountUpdate.jsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react'
+import { BaseUpdate, FormContainer, FormSection, JsonField, NotFound } from 'features/shared/components'
+import { reduxForm } from 'redux-form'
+
+class Form extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.submitWithErrors = this.submitWithErrors.bind(this)
+
+    this.state = {}
+  }
+
+  submitWithErrors(data) {
+    return this.props.submitForm(data, this.props.item.id).catch(err => {
+      throw {_error: err}
+    })
+  }
+
+  componentDidMount() {
+    this.props.fetchItem(this.props.params.id).then(resp => {
+      if (resp.items.length == 0) {
+        this.setState({notFound: true})
+      }
+    })
+  }
+
+  render() {
+    if (this.state.notFound) {
+      return <NotFound />
+    }
+    const item = this.props.item
+
+    if (!item) {
+      return <div>Loading...</div>
+    }
+
+    const {
+      fields: { tags },
+      error,
+      handleSubmit,
+      submitting
+    } = this.props
+
+    const title = <span>
+      {'Edit account tags '}
+      <code>{item.alias ? item.alias :item.id}</code>
+    </span>
+
+    const hint = <div>
```

Comment:
I notice that the hint disappears when there is an error (e.g. invalid JSON). I think the hint should persist regardless of errors. We may have to change it from a hint to a div that captions the whole form, rather than a single field?